projects/storefrontlib/cms-structure/routing/default-routing-config.ts

```diff
@@ -93,7 +95,7 @@ export const defaultRoutesConfigFactory: () => RoutingConfig = () => {
    * Configuration necessary to allow customization of login form path,
    * which have to be the same as configured in oauth client
    */
-  if (featureToggles.authorizationCodeFlowByDefault) {
+  if (featureToggles.nativeSupportForRedirectingOAuthFlows) {
```

Comment:
This change is related to the enhancements we made to support oAuth flows involving a redirect.

At the very least, it is related to custom login, and should be added when the feature is enabled.